pkg/resources/database/testdata/datasource.tmpl

```diff
@@ -2,7 +2,4 @@ data "exoscale_database_uri" "{{ .ResourceName }}" {
 	name = {{ .Name }}
 	type = "{{ .Type }}"
 	zone = "{{ .Zone }}"
-    timeouts {
-        read = "20m"
-    }
```

Comment:
I guess this shouldn't be part of this PR ?

Reply:
You are right, it is not in the scope of the fix.